docs(coral): Make running Coral inside Klaw vs in dev mode more clear (#322)
aiven/klaw: coral/README.md

## Installation and usage

** ℹRequirements**
- [node](https://nodejs.org/en/) needs to be installed (see [npmcr](.npmrc) for version).
- Coral uses [pnpm](https://pnpm.io/) (version 7) as a package manager. Read their official documentation [how to install](https://pnpm.io/installation) pnpm. 

### Usage: How to run Coral inside the Klaw application

1. navigate to this directory
2. run `pnpm install`
3. run `make enable-coral-in-springboot` (see our [Makefile](Makefile)). This will enable Coral in Klaw. It also moves the Coral build files to the right directory.
4. go to the root directory and follow the [instructions on how to run Klaw](../README.md#Install)

> ➡️ Dependent on your Springboot [application properties](https://github.com/aiven/klaw/blob/main/core/src/main/resources/application.properties#L5) in Springboot: 

Reviewer: ```suggestion
➡️ Based on Springboot [application properties](https://github.com/aiven/klaw/blob/main/core/src/main/resources/application.properties#L5) configuration: 
```

> - Klaw will run in `https://localhost:9097` if certifications are enabled

Reviewer: ```suggestion
- Klaw will run in `https://localhost:9097` if TLS is abled
```

> ℹ️ Coral uses `pnpm run` as a package manager. Read their official documentation [how to install](https://pnpm.io/installation) it.

Reviewer: ```suggestion
ℹ️ Coral uses `pnpm run` as a package manager. Read their official documentation [how to install](https://pnpm.io/installation) pnpm.
```

Reviewer: Any specific versions of node/pnpm are required to run klaw

Author: We're running the lasted PNPM, that is be installed automatically if users install it new. But I can make a note about what major version we're using at least. 

The node version is set per `.npmrc`, so users would get notified if they need a different node version. I will add that in the docs and link the npmrc file, then it's easy to keep the versions in sync!

### Usage: How to run Coral inside the Klaw application

1. navigate to this directory
2. run `pnpm install`

> 3. run `make enable-coral-in-springboot` (see our [Makefile](Makefile)). This will enable Coral in Klaw. It also add the current Coral build files in the right place.

Reviewer: ```suggestion
3. run `make enable-coral-in-springboot` (see our [Makefile](Makefile)). This will enable Coral in Klaw. It also moves the Coral build files to the right directory.
```

4. go to the root directory and follow the [instructions on how to run Klaw](../README.md#Install)

> ➡️ Klaw will run in `http://localhost:9097` (note: it's `http` instead of `https`!)

Author: Can you tell me where they can see this? (`applications.properties` ?). As a no-Java-experience user :D I didn't know that. 
So it may be helpful to add that in case that someone wants to contribute on the FE and run Klaw, but is also missing that background.

> ➡️ Klaw will run in `http://localhost:9097` (note: it's `http` instead of `https`!)

Reviewer: If they enable these properties, server would run on https

> We use the component library of Aiven's design system:

Reviewer: ```suggestion
Klaw uses/depends on the component library of Aiven's design system:
```
